man/tpm2_pcrallocate.1.md

```diff
@@ -0,0 +1,57 @@
+% tpm2_pcrallocate(1) tpm2-tools | General Commands Manual
+%
+% MARCH 2019
+
+# NAME
+
+**tpm2_pcrallocate**(1) - Change the allocated PCRs of a TPM
+
+# SYNOPSIS
+
+**tpm2_pcrallocate** [*OPTIONS*] [*ALLOCATION*]
+
+# DESCRIPTION
+
+**tpm2_pcrallocate**(1) - allows the user to specify an allocation for the TPM.
+If no *ALLOCATION* is given, then SHA1 and SHA256 banks with PCRs 0 - 23 are
+allocated.
+
+# ALLOCATION
+
+A list of banks and selected pcrs. To allocate a new bank a hash algorithm followed
+by a colon is presented, `sha1:` followed by the keyword `all` or a space separated
+list of pcrs; e.g. `sha1: all` or `sha256: 0 1 5 15 16`. Multiple of these bank
+definitions can be presented sequentially; e.g. `sha1: all sha256: all`. If no
+allocation is presented then `sha1: all sha256: all` is set.
+
+# OPTIONS
+
+  * **-P**, **--auth-platform**=_AUTH\_HIERARCHY_\VALUE_:
+
+    Optional authorization value. Authorization values should follow the
+    "authorization formatting standards", see section "Authorization Formatting".
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+[context object format](common/ctxobj.md)
+
+[authorization formatting](common/authorizations.md)
+
+# EXAMPLES
+
+## To allocate the two default banks
+```
+tpm2_pcrallocate
+```
+## To make a custom allocation with a platform authorization
+```
+tpm2_pcrallocate -P abc sha1: 7 8 9 10 16 17 18 19  sha256: 0 1 2 3 4 5 6
```

Comment:
cant you use the -L style parsing code for pcrlist here?

Reply:
Will do... I'll extend -L style parsing by the all keyword though.

Reply:
Done